WoW-Realm-Tracker/ViewControllers/FavoritesViewController.swift

```diff
@@ -83,8 +84,8 @@ class FavoritesViewController: UITableViewController {
 }
 
 extension FavoritesViewController: RealmsDelegate {
-    func receivedRealms(realms: [Realm]) {
-        controller = FavoriteRealmsController(realms: realms)
+    func receivedRealms(response: Result<[Realm], NSError>) {
+        response.map { realms in self.controller = FavoriteRealmsController(realms: realms) }
```

Comment:
I like the use of `map` here. Should the `tableView.reloadData()` also go within the ` {} ` here?